### PR TITLE
[ES] Spanish Showdown 2022

### PR DESCRIPTION
**Pull request by fajar13k**
## Self-check

- [X] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

## Description

- Spanish translation of Spanish Showdown 2022 article
- Spanish Showdown 2021 logo replacement

### CODE REVIEW

File: wiki/Tournaments/SS/2022/es.md [hunk 1]
```diff
@@ -0,0 +1,631 @@
...
+  1. [Vaundy - Hadaka no Yuusha (AirinCat) \[Insane\]](https://osu.ppy.sh/beatmapsets/1669880#osu/3410660)
+  2. [Magnetude & Gridoptic - At All Costs (feat. Julia Marks) (Zer0-) \[Insane\]](https://osu.ppy.sh/beatmapsets/1108285#osu/2567077)
+  3. [-45 - Midorigo Queen Bee (PandaHero) \[Mirash's Insane\]](https://osu.ppy.sh/beatmapsets/812792#osu/1716393)
+  4. [Tainaka Sachi - Akio (Shurelia) \[Life is so sweet!\]](https://osu.ppy.sh/beatmapsets/1287052#osu/2672123)
+- Tiebreaker
+  1. **[Fuki - Sacred Bones Riot (Scarsnic) \[Consecration\]](https://osu.ppy.sh/beatmapsets/992991#osu/2076544)**
+
+### Octavos de final
+
+*El pack de mapas no disponible.*
```
Comment (Repflez) [inline wiki/Tournaments/SS/2022/es.md:215]:
```suggestion
*El pack de mapas no está disponible.*
```

File: wiki/Tournaments/SS/2022/es.md [hunk 2]
```diff
@@ -0,0 +1,631 @@
...
+  - SS
+  - SS2022
+  - SS 2022
+---
+
+# Spanish Showdown 2022
+
+![Logo del SS 2022](img/logo.jpg)
+
+El **Spanish Showdown 2022** (***SS 2022***) fue un torneo español de doble eliminación 1vs1 de osu! organizado por ::{ flag=ES }:: [E R N E S T O](https://osu.ppy.sh/users/2127413), ::{ flag=ES }:: [M A N O L O](https://osu.ppy.sh/users/12296128), y ::{ flag=ES }:: [Margo](https://osu.ppy.sh/users/6995685). El torneo iba dedicado a los mejores jugadores de osu! españoles, sin límite en su rango. Ha sido la segunda edición del Spanish Showdown.
```
Comment (Repflez) [inline wiki/Tournaments/SS/2022/es.md:12]: > El torneo iba dedicado a los mejores jugadores de osu! españoles, sin límite en su rango. Ha sido la segunda edición del Spanish Showdown.

Is this supposed to be in past tense?

**Review by Repflez — CHANGES_REQUESTED** [review]
Just some nitpicks from me, but otherwise, LGTM

File: wiki/Tournaments/SS/2022/es.md (hunk 2 above)
Comment (fajar13k) [reply wiki/Tournaments/SS/2022/es.md:12]: Correct, tournament already ended.
inb4 another wave of suggestions.

**Review by Repflez — APPROVED** [review]

**Pull request merged**